lib/ret_web/channels/hub_channel.ex

```diff
@@ -372,14 +379,18 @@ defmodule RetWeb.HubChannel do
     if account |> can?(update_hub(hub)) do
       name_changed = hub.name != payload["name"]
       member_permissions_changed = hub.member_permissions != payload |> Hub.member_permissions_from_attrs()
+      can_change_privacy = account |> can?(update_hub_privacy(hub))
+      privacy_changed = can_change_privacy and hub.privacy != payload["privacy"]
```

Comment:
will `hub.privacy` be a string or an atom here? I forget how it comes out of the database into the record (i'm guessing you tested this but just checking)